integreat_cms/core/logging_filter.py

```diff
@@ -0,0 +1,16 @@
+import logging
+
+from django.conf import settings
+
+
+class DebugRequestFilter(logging.Filter):
+    def filter(self, record):
+        if settings.REQUEST_DEBUG_ADMIN_IPS:
+            from integreat_cms.core.middleware.debug_request import get_request
+
+            request = get_request()
+            return record.levelno >= logging.INFO or (
+                request is not None
+                and request.user.username in settings.REQUEST_DEBUG_USERS
```

Comment:
@PeterNerlich now that we have the request object we can actually use the usernames instead of IPs. That is much more convenient. We can even enabled debug logging for region editors if we cannot reproduce the bug ourselves.

Reply:
Alternatively, we could add a setting to the user preferences. However, I would actually prefer to solve this on the server side. `systemctl reload apache2` should work as well and should not be disruptive.